### PR TITLE
bump golang extension to 0.16.1

### PR DESCRIPTION
**Pull request by gattytto**
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

Closes https://github.com/eclipse/che/issues/17665

### CODE REVIEW

File: v3/plugins/golang/go/0.16.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+name: go
+version: 0.16.1
+type: VS Code extension
+displayName: Go
+title: Rich Go language support for Visual Studio Code
+description: This extension provides rich language support for the Go programming language in VS Code.
+icon: https://raw.githubusercontent.com/golang/vscode-go/v0.16.1/images/go-logo-blue.png
+repository: https://github.com/golang/vscode-go
+category: Language
+firstPublicationDate: '2020-06-08'
```
Comment (svor) [inline v3/plugins/golang/go/0.16.1/meta.yaml:12]: firstPublicationDate should be updated

**Review by svor — APPROVED** [review]
I tested on https://che.openshift.io/ and everything works good! Just one comment about **firstPublicztionDate**

**Comment by gattytto** [comment]
image
...

**Comment by gattytto** [comment]
image

**Comment by ericwill** [comment]
> image
> ...

Huh, how come this failed all of a sudden?

**Comment by gattytto** [comment]
> > image
> > ...
> 
> Huh, how come this failed all of a sudden?

has to be in the footer

**Pull request closed without merging**